### ISSUE TITLE
Eslint: Unable to resolve path to module 'vite-plugin-ssr/plugin' as of 0.3.57

### ISSUE THREAD

**Issue by kamui** [brillout/vite-plugin-ssr#270]
As of version 0.3.57:

in `vite.config.ts`:
```
import { ssr } from "vite-plugin-ssr/plugin"
```

results in:

```
Unable to resolve path to module 'vite-plugin-ssr/plugin'
```

and anywhere I call `getPage`:

```
import { getPage } from "vite-plugin-ssr/client"
```

results in:

```
Unable to resolve path to module 'vite-plugin-ssr/client'
```

This doesn't happen in 0.3.56, but probably related to adding ESM support?

**Comment by brillout** [comment]
What's your Node.js version?

**Comment by brillout** [comment]
Also, I need a full stack trace.

**Comment by kamui** [comment]
Node version is: `v16.13.2`

There really isn't a stacktrace since it's eslint that fails, but here's that:

```
> eslint .


./renderer/_default.page.client.tsx
  6:25  error  Unable to resolve path to module 'vite-plugin-ssr/client'  import/no-unresolved

./vite.config.ts
  5:21  error  Unable to resolve path to module 'vite-plugin-ssr/plugin'  import/no-unresolved

✖ 2 problems (2 errors, 0 warnings)
```

**Comment by brillout** [comment]
I see. I'll write the fix today/tomorrow.

**Comment by brillout** [comment]
Fix released in `v0.3.59`. It should now work but I didn't test it; let me know if you still have problems with eslint.
